tests/e2e-pw/playwright-utils/test.ts

```diff
@@ -0,0 +1,145 @@
+/**
+ * External dependencies
+ */
+import { test as base, expect, request } from '@playwright/test';
+import type { ConsoleMessage } from '@playwright/test';
+import {
+	Admin,
+	Editor,
+	PageUtils,
+	RequestUtils,
+} from '@wordpress/e2e-test-utils-playwright';
+
+import { TemplateApiUtils, STORAGE_STATE_PATH } from '@woocommerce/e2e-utils';
+
+/**
+ * Set of console logging types observed to protect against unexpected yet
+ * handled (i.e. not catastrophic) errors or warnings. Each key corresponds
+ * to the Playwright ConsoleMessage type, its value the corresponding function
+ * on the console global object.
+ */
+const OBSERVED_CONSOLE_MESSAGE_TYPES = [ 'warn', 'error' ] as const;
+
+/**
+ * Adds a page event handler to emit uncaught exception to process if one of
+ * the observed console logging types is encountered.
+ *
+ * @param message The console message.
+ */
+function observeConsoleLogging( message: ConsoleMessage ) {
```

Comment:
Heads-up that there is a JSDoc warning here.

Reply:
Fixed with 21b6eb63c779ab64ea0dd4f78e825ab16a206abc